src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchema.cs

```diff
@@ -88,35 +89,41 @@ public class XmlSchema : XmlSchemaObject
             return parser.XmlSchema;
         }
 
+        [RequiresUnreferencedCode(XmlSerializer.LinkerSerializationMessage)]
```

Comment:
I think your changes to the corresponding `ref` file were lost in the latest commits.